tests/paddle/test_operators.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# See LICENSE for license information.
+"""Test TE operators"""
+
+import pytest
+import paddle
+from utils import assert_allclose, create_fp8_meta
+
+import transformer_engine    # pylint: disable=unused-import
+import transformer_engine_paddle as tex    # pylint: disable=wrong-import-order
+
+from transformer_engine.paddle.cpp_extensions import cast_to_fp8, cast_from_fp8, gemm, fp8_gemm
+from transformer_engine.paddle.fp8 import is_fp8_available
+
+paddle.seed(10)
+GEMM_CASES = [(256, 256, 512), (32, 32, 32), (16384, 1024, 2816), (16384, 2816, 1024),
+              (16384, 1024, 1024)]
+is_fp8_supported, reason = is_fp8_available()
+
+
+def test_quantize_dequantize():
+    """
+    Test cast_to_fp8 and cast_from_fp8
+    """
+    a = paddle.rand(shape=(32, 32), dtype='float32')
+    # Init fp8_meta
+    fp8_meta = create_fp8_meta(num_fp8_tensors=3, amax_history_len=10)
+    for fp8_dtype in [tex.DType.kFloat8E4M3, tex.DType.kFloat8E5M2]:
+        a_fp8 = cast_to_fp8(a, fp8_meta, tex.FP8FwdTensors.GEMM1_OUTPUT, otype=fp8_dtype)
+        b = cast_from_fp8(a_fp8,
+                          fp8_meta,
+                          tex.FP8FwdTensors.GEMM1_OUTPUT,
+                          itype=fp8_dtype,
+                          otype=tex.DType.kFloat32)
+        assert_allclose(a, b, rtol=5e-2, atol=5e-2)
+
+
+class TestGemm:
+    """
+    Tests for gemm(cuBLASLt) operator
+    """
+
+    @staticmethod
+    @pytest.mark.parametrize('m,n,k', GEMM_CASES)
+    def test_bf16(m, n, k):
```

Comment:
This test (and `test_bf16_inplace`) are failing on a V100. I propose changing it to fp32.

Reply:
I have added skip conditions for these tests. I think it is better to keep the BF16 tests, as we mainly target at BF16 and FP8 dtype for TE operators. We will use paddle native operators for FP32 computations.